docs/index.rst

```diff
@@ -9,23 +9,9 @@
-           :target: https://mybinder.org/v2/gh/scivision-gallery/scivision-basic-usage-examples/HEAD?labpath=how-to-use-scivision.ipynb
-
-- Browse through a gallery of interactive notebooks from research projects using Scivision at https://github.com/scivision-gallery
-
--  The `Scivision webapp <https://alan-turing-institute.github.io/scivision/>`_ is a tool for both using and contributing to scivision.
-         
-         > Note: The webapp is a work in progress and there will be more documentation soon. In the meantime feel free to open any bug or feature request issues at https://github.com/alan-turing-institute/scivision.
+This website hosts the documentation for the `scivision` Python package.
 
+To learn more about Scivision, visit `our main project website <https://alan-turing-institute.github.io/scivision/>`_.
```

Comment:
Perhaps, '...if you are knew to the project', which I think is the subtext of this

Reply:
yep agreed, updated